src/sass/extends/_blockquotes.scss

```diff
@@ -145,7 +149,11 @@ blockquote:before {
     img {
       max-width: 180px;
       height: auto;
+      background-repeat: no-repeat;
+      background-position: 50%;
       border-radius: 50%;
+      width: 180px;
```

Comment:
- https://github.com/ASU/asu-unity-stack/pull/296 for background.
- The "fix" here breaks the mobile display of block quotes by imposing a set height and width on the image. (Which should be responsive.)
- The intent is to allow non-square images to be included in the markup... but this falls short of that fix. 
- We have not implemented the new markup for Blockquotes in UDS-Wordpress as of yet, so we're still OK to merge this even though it's wrong.